modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-0_{context}"]
+= Release Notes for Red Hat {pipelines-title} Technology Preview 1.0
+
+[id="new-features-1-0_{context}"]
+== New features
+* {pipelines-title} 1.0 is available on {product-title} 4.4 and newer versions on channels *ocp-4.4.0* and *preview*.
+* {op-title} 1.0 has been updated to use :
+** Tekton Pipelines 0.11.1
+** Tekton `tkn` CLI 0.9.0
+** Tekton Triggers 0.4.0
+** ClusterTasks based on Tekton Catalog 0.11
+
+
+[id="fixed-issues-1-0_{context}"]
+== Fixed issues
+* link:https://issues.redhat.com/browse/SRVKP-637[SRVKP-637] : Previously, the `DeploymentConfig` Task triggered a new deployment build even when an image build was already in progress. This caused the deployment in Pipeline to fail. With this fix, the `deploy task` command  is now replaced with the `oc rollout status` command which waits for the in-progress deployment to finish.
+* link:https://issues.redhat.com/browse/SRVKP-631[SRVKP-631]: Support for `APP_NAME` parameter is now added in Pipeline templates.
+* link:https://issues.redhat.com/browse/SRVKP-616[SRVKP-616]: Pipeline template for Java S2I failed to look up image in the registry. With this fix, the look up will now be performed using the existing image Pipeline Resources instead of user provided `IMAGE_NAME` parameter.
+* link:https://issues.redhat.com/browse/SRVKP-383[SRVKP-383]: All the Openshift Pipelines images are now based on the Red Hat Universal Base Images (UBI).
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+* If you are upgrading from an older version of {pipelines-title}, you will need to delete your existing deployments before upgrading to version 1.0.
```

Comment:
What does this mean? They should uninstall the operator? delete the `Config` CR? delete the pipelines and tasks?

Reply:
@siamaksade - Should we direct the user to the uninstall operator procedure , as specified in this PR - https://github.com/nkakkar81/openshift-docs/blob/526797737fee955ac2b894b0990c0ba0abbc3361/openshift_pipelines/uninstalling-pipelines.adoc

Reply:
Suggest: ...you will need to delete your existing deployments/you will need to delete the Custom Resources (CRs) that were added by default when you installed the OpenShift Pipelines Operator and then uninstall the OpenShift Pipelines Operator. For more details see <link>

Reply:
Have modified the description PTAL